velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -995,5 +996,298 @@ TEST_F(StringTest, trim) {
       trimWithTrimStr("\u6570", "\u6574\u6570 \u6570\u636E!"),
       "\u6574\u6570 \u6570\u636E!");
 }
+
+TEST_F(StringTest, split) {
+  auto limit = -1;
+  auto delim = ",";
+  auto numRows = 3;
+  auto data = std::vector<std::string>{
+      {"I,he,she,they"}, // Simple
+      {"one,,,four,"}, // Empty strings
+      {""}, // The whole string is empty
+  };
+
+  auto createStrings = [&](RowSet /*rows*/) {
```

Comment:
rows are not in use, can we create the string vector without it like below?
> makeFlatVector<StringView>({"ALGERIA", "ARGENTINA"})

Reply:
can't, we need leverage `row` to init with `data[row]`